cocos/ui/UIEditBox/iOS/CCUIEditBoxIOS.h

```diff
@@ -59,4 +59,7 @@
 - (void)openKeyboard;
 - (void)closeKeyboard;
 
+- (void)setInnerPadding:(float)left :(float)top :(float)right :(float)bottom;
+- (void)setTextAlignMent:(cocos2d::TextHAlignment)hAlign :(cocos2d::TextVAlignment)vAlign;
```

Comment:
__[Minor]:__ it would be appreciated if you add names to each parameter, since it follows apple's code guidelines:

```objc
- (void)setInnerPaddingLeft:(float)left top:(float)top right:(float)right bottom:(float)bottom;
- (void)setTextHorizontalAlignment:(cocos2d::TextHAlignment)hAlign verticalAlignment:(cocos2d::TextVAlignment)vAlign;
```